docs/storage/globus.md

```diff
@@ -15,6 +15,15 @@ To get started, open your browser to <https://app.globus.org/collections/gcp?gen
 
 You will setup a new collection, i.e. endpoint. Enter a **Collection Display Name** that is memorable to you, e.g. "Cluster Dirs", and select **Generate Setup Key**.
 
+As of 5/1/2024, we have identified an issue with Globus producing empty keys with the above step, and they have been notified and are working on the issue.
+The workaround is to create the collection and get the setup key via the globus CLI.  Run the following in a SSH session on a RCC cluster login node:
+
+```bash
+module load globusconnect
+globus login
+globus gcp create mapped "Cluster Dirs"
+ ```
```

Comment:
The space in front of this code block closing is causing the section to not render correctly.